<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UVBuild</class>
 <widget class="QDockWidget" name="UVBuild">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>476</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>257</width>
    <height>209</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>生成光照图UV</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QRadioButton" name="m_useStretch">
        <property name="text">
         <string>变形量上限</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="m_texHeight">
        <property name="text">
         <string>512</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>图表间距：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>纹理宽：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="m_texWidth">
        <property name="text">
         <string>512</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>纹理高：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="m_maxStretch">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="m_useChart">
        <property name="text">
         <string>图表数量上限</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="m_maxCharts">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="m_gutter">
        <property name="maximum">
         <number>512</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="m_generate">
        <property name="text">
         <string>生成</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_restore">
        <property name="text">
         <string>还原</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>m_generate</tabstop>
  <tabstop>m_texWidth</tabstop>
  <tabstop>m_texHeight</tabstop>
  <tabstop>m_gutter</tabstop>
  <tabstop>m_useStretch</tabstop>
  <tabstop>m_maxStretch</tabstop>
  <tabstop>m_useChart</tabstop>
  <tabstop>m_maxCharts</tabstop>
  <tabstop>m_restore</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_useStretch</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_maxStretch</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_useChart</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_maxCharts</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
